### PR TITLE
Add storybook story for the ToolbarButton component

### PR DESCRIPTION
**Pull request by brentswisher**
## Description
Adds a storybook story for the ToolbarButton component as part of #17973

## How has this been tested?
* run `npm run storybook:dev`
* See the new ToolbarButton story under components

## Screenshots
Components___ToolbarButton_-_Default_⋅_Storybook

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by gziolo** [comment]
I guess it’s one on those PRs that fails because of StoryShots 😅

File: packages/components/src/toolbar-button/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Internal dependencies
+ */
+import { Toolbar } from '../../';
+import ToolbarButton from '../';
+
+export default { title: 'Components/ToolbarButton', component: ToolbarButton };
+
+export const _default = () => {
+	const label = text( 'Label', 'This is an example label.' );
+	const icon = text( 'Icon', 'ellipsis' );
```
Comment (gziolo) [inline packages/components/src/toolbar-button/stories/index.js:16]: Can you pick a different icon, please? 😂

**Review by gziolo — APPROVED** [review]

File: packages/components/src/toolbar-button/stories/index.js (hunk 1 above)
Comment (brentswisher) [reply packages/components/src/toolbar-button/stories/index.js]: How about the WordPress logo?

Comment (gziolo) [reply packages/components/src/toolbar-button/stories/index.js]: 💘

**Pull request merged**